discovery/interface.go

```diff
@@ -87,9 +39,11 @@ var ErrPresentationRegistrationFailed = errors.New("registration of Verifiable P
 type Server interface {
 	// Register registers a presentation on the given Discovery Service.
 	// If the presentation is not valid, or it does not conform to the Service ServiceDefinition, it returns an error.
-	Register(serviceID string, presentation vc.VerifiablePresentation) error
-	// Get retrieves the presentations for the given service, starting at the given timestamp.
-	Get(serviceID string, startAt *Tag) ([]vc.VerifiablePresentation, *Tag, error)
+	// If the serviceID is not configured as server definition, the call will be forwarded to the configured server.
+	Register(context context.Context, serviceID string, presentation vc.VerifiablePresentation) error
+	// Get retrieves the presentations for the given service, starting from the given Timestamp.
+	// If the serviceID is not configured as server definition, the call will be forwarded to the configured server.
+	Get(context context.Context, serviceID string, startAt int) (map[string]vc.VerifiablePresentation, int, error)
```

Comment:
in the implementation this is called `startAfter`. Is there an `off-by-one` bug here?

The store does `timestamp > :startAfter`